fidesops.toml

```diff
@@ -44,3 +45,6 @@ analytics_opt_out = false
 
 [admin_ui]
 enabled = true
+
+[notifications]
+send_request_completion_notification = true
```

Comment:
Do we need an easier way to disable this during local dev? We don't want to be sending random emails unintentionally.  It seems like in dev mode the default should be to not send.

Reply:
It makes sense to instead default to false in our own `fidesops.toml`, so I'll update!